ui/components/atoms/A005-Tab/index.js

```diff
@@ -0,0 +1,24 @@
+// List Libraries being used
+import React from 'react'
+import createPropTypes from 'json-schema-prop-types'
+
+import Tab from '@material-ui/core/Tab'
+
+// Import JSON schema
+import schema from './schema'
+
+// Import styles here - see example setup for styles at ui/example/components/atoms/checkbox
+import Icon from '../../particles/P001-Icon'
+
+// Define Atom as JSX
+const A005 = props => {
+  return (
+    <Tab icon={props.icon} />
```

Comment:
My understanding was that we would be passing in an <Icon iconname=props.iconname/> here?

Reply:
String

Reply:
Dan, I'm working on this right now